pipeline-runner/R/gem2s-2-load_user_files.R

```diff
@@ -239,6 +209,38 @@ parse_rhapsody_matrix <- function(config, input_dir) {
   return(list(counts_list = counts_list, annot = annot))
 }
 
+read_10x_annotations <- function(annot_fpath, sample) {
+  gene_column <- 1
```

Comment:
maybe change to "seurat_rownames_col" or add comment explaining

Reply:
Named this way because the read10x parameter has this name